_includes/facebook_logo.html

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="22pt" height="22pt" viewBox="0 0 22 22" version="1.1">
-<g id="surface1">
-<path style=" stroke:none;fill-rule:nonzero;fill:rgb(23.137255%,34.901961%,59.607843%);fill-opacity:1;" d="M 20.71875 21.925781 C 21.386719 21.925781 21.925781 21.386719 21.925781 20.722656 L 21.925781 1.28125 C 21.925781 0.613281 21.386719 0.0742188 20.71875 0.0742188 L 1.28125 0.0742188 C 0.613281 0.0742188 0.0742188 0.613281 0.0742188 1.28125 L 0.0742188 20.722656 C 0.0742188 21.386719 0.613281 21.925781 1.28125 21.925781 Z M 20.71875 21.925781 "/>
-<path style=" stroke:none;fill-rule:nonzero;fill:rgb(100%,100%,100%);fill-opacity:1;" d="M 15.152344 21.925781 L 15.152344 13.464844 L 17.992188 13.464844 L 18.417969 10.164062 L 15.152344 10.164062 L 15.152344 8.058594 C 15.152344 7.105469 15.417969 6.453125 16.785156 6.453125 L 18.53125 6.453125 L 18.53125 3.503906 C 18.230469 3.464844 17.195312 3.375 15.988281 3.375 C 13.46875 3.375 11.746094 4.910156 11.746094 7.734375 L 11.746094 10.164062 L 8.898438 10.164062 L 8.898438 13.464844 L 11.746094 13.464844 L 11.746094 21.925781 Z M 15.152344 21.925781 "/>
-</g>
+<svg width="22pt" height="22pt" version="1.1" viewBox="0 0 22 22" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I don't think width and height attributes are needed, you should be able to remove them and keep only viewBox